packages/grafana-ui/README.md

```diff
@@ -16,4 +16,4 @@ See [package source](https://github.com/grafana/grafana/tree/main/packages/grafa
 
 ## Development
 
-For development purposes we suggest using `yarn link` that will create symlink to @grafana/ui lib. To do so navigate to `packages/grafana-ui` and run `yarn link`. Then, navigate to your project and run `yarn link @grafana/ui` to use the linked version of the lib. To unlink follow the same procedure, but use `yarn unlink` instead.
+For development purposes we suggest using `yarn link` that will create symlink to @grafana/ui lib. To do so navigate to `packages/grafana-ui` and run `YARN_IGNORE_PATH=1 yarn link`. Then, navigate to your project and run `yarn link "@grafana/ui"` to use the linked version of the lib. To unlink follow the same procedure, but use `yarn unlink` instead.
```

Comment:
Where does `YARN_IGNORE_PATH=1` come from?

Reply:
From me poking yarn with a stick until I was able to get `yarn link` to actually do something.

I got it from here:
https://classic.yarnpkg.com/lang/en/docs/cli/link/
`In case if you want to link yarn 3+ project into project which uses older versions of yarn`

I noticed that when running `yarn link` in the `packages/grafana-ui` directory, nothing was added to my `~/.config/yarn/link/` directory, but running `YARN_IGNORE_PATH=1 yarn link` returns the proper output, and success message:
```
galen@gtk grafana-ui % YARN_IGNORE_PATH=1 yarn link                                    
yarn link v1.22.19
success Registered "@grafana/ui".
info You can now run `yarn link "@grafana/ui"` in the projects where you want to use this package and it will be used instead.
✨  Done in 0.03s.
```
vs

```
galen@gtk grafana-ui % yarn link
[... truncated for brevity]
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed in 0s 418ms
➤ YN0000: · Done with warnings in 1s 323ms
```

Reply:
Nice find!